_docs/welcome/feature-stages.md

```diff
@@ -29,94 +28,75 @@ within the product, not to the product as a whole. Here is a high level descript
 |  **Performance**         | Not quantified or guaranteed     | Not quantified or guaranteed         | Perf (latency/scale) is quantified, documented, with guarantees against regression 
 |   **Deprecation Policy**        | None     | Weak - 3 months         | Dependable,  Firm. 1 year notice will be provided before changes
 
-## Istio 0.3.0 features status
+## Istio 0.4 features
```

Comment:
Should we drop the 0.4 here and just have this be a continually up-to-date version?  Or do we want to snapshot this each release?

Reply:
I would say we follow the former.. i.e. the release page will have a feature's status, for the current release.